swap/src/cli/api/request.rs

```diff
@@ -1216,133 +1259,261 @@ pub async fn resolve_approval_request(
     Ok(ResolveApprovalResponse { success: true })
 }
 
-fn qr_code(value: &impl ToString) -> Result<String> {
-    let code = QrCode::new(value.to_string())?;
-    let qr_code = code
-        .render::<unicode::Dense1x2>()
-        .dark_color(unicode::Dense1x2::Light)
-        .light_color(unicode::Dense1x2::Dark)
-        .build();
-    Ok(qr_code)
-}
+pub async fn fetch_quotes_task(
+    rendezvous_points: Vec<Multiaddr>,
+    namespace: XmrBtcNamespace,
+    sellers: Vec<Multiaddr>,
```

Comment:
use the sellers that we pass in here, pass into list_sellers_init or let fetch_fn take a Vec<Multiaddr>